docs/api.md

```diff
@@ -4,10 +4,18 @@ Popover attaches to an element and displays on hover/blur.
 
 ## Properties
 
-| Property    | Attribute   | Type     | Default | Description                                      |
-|-------------|-------------|----------|---------|--------------------------------------------------|
-| `for`       | `for`       | `String` |         | Defines an `id` for an element in the DOM to trigger on hover/blur. |
-| `placement` | `placement` | `String` | "top"   | Expects top/bottom - position for popover in relation to the element. |
+| Property           | Attribute          | Type      | Default | Description                                      |
+|--------------------|--------------------|-----------|---------|--------------------------------------------------|
+| `for`              | `for`              | `String`  |         | Defines an `id` for an element in the DOM to trigger on hover/blur. |
+| `isPopoverVisible` | `isPopoverVisible` | `boolean` | false   | Boolean for if popover is visible or not.        |
+| `isSticky`         | `isSticky`         | `boolean` |         | If true, when user clicks trigger, the popover will persist its visibility. If false, popover will disappear when mouseout over trigger. |
+| `placement`        | `placement`        | `String`  | "top"   | Expects top/bottom - position for popover in relation to the element. |
+
+## Methods
+
+| Method   | Type       | Description                      |
+|----------|------------|----------------------------------|
+| `toggle` | `(): Void` | Toggles the popover's open state |
```

Comment:
This method also should be hidden from the public API docs.

Reply:
Yeah, this was my bad. I didn't realize that this needed to be used in a public space.